README.md

```diff
@@ -125,7 +126,7 @@ rooms, issue trackers, mailing lists, social media.
 * Close relationship between project members and users or customers can be a key to success.
 
 ### What can people learn?
-* See how people use your team's solutions.
+* Learn how people use your team's solutions.
```

Comment:
I would maybe stick with the previous version to not repeat the `learn` from the question

Reply:
oh, that's a good point

originally I replaced it with `Observe` because I felt that `See` is too generic, but that felt too formal and rigid, so I switched to `Learn` and didn't read the whole context

will change it back to `See`